tests/unit_tests/test_pychop.py

```diff
@@ -158,7 +186,7 @@ def test_fermi_moderator_width_analytical(e_init, pychop_fermi_data):
     _test_moderator_width_analytical(e_init, *pychop_fermi_data, PyChopModelFermi)
 
 
-@pytest.mark.parametrize('e_init', EINIT)
+@pytest.mark.parametrize('e_init', EINIT_NONFERMI, ids=[f'ei={ei}' for ei in EINIT_NONFERMI])
```

Comment:
Instead of the list comprehension here, we can provide a function that will be evaluated on each set of parameter values. I did this a whole bunch in the testing PR, so there may be some merge conflicts to sort out one way or the other.